vital/types/detected_object.h

```diff
@@ -43,10 +43,11 @@
 #include <vector>
 
 #include <vital/types/detected_object_type.h>
```

Comment:
If you wind up making other changes, I usually re-sort existing headers also when changing an include list. (`detected_object_type.h` seems to be the only one out of place. If you intend for it to be "file local" as opposed to just "same library", that's okay too, but then it should have a blank line separating it from the others.)